gapis/api/templates/api.go.tmpl

```diff
@@ -415,6 +415,19 @@ import (
     }
   }
 
+func (m {{$name}}) DestroyHandlesʷ(ϟctx context.Context, ϟw ϟapi.StateWatcher) {
+    if m.Map != nil {
+      if ϟw != nil {
+        ϟw.OnSet(ϟctx, m, ϟapi.CompleteFragment{},
+          ϟapi.NilReference{}, ϟapi.NilReference{})
+        for key, _ := range (*m.Map) {
+          ϟw.CloseForwardDependency(ϟctx, key)
+        }
+      }
+      *m.Map = make(map[{{$key}}]{{$value}})
+    }
+  }
+
```

Comment:
I think these should now be correct implementations of the dependency graph side of DestroyHandles and Clear.

However, there will be minor conflicts with some incoming dependency graph changes, depending on which PR goes in first.

Reply:
Thanks!